src/main/java/io/supertokens/webserver/InputParser.java

```diff
@@ -137,7 +137,12 @@ public static String parseStringOrThrowError(JsonObject element, String fieldNam
             if (!stringified.contains("\"")) {
                 throw new Exception();
             }
-            return ((JsonObject) element).get(fieldName).getAsString();
```

Comment:
This function is called for non GET request. We might want to add the trim and toLowerCase (in case of email), to the GET request equivalent as well. The functions are `getQueryParamOrThrowError` and `getCommaSeparatedStringArrayQueryParamOrThrowError`

Reply:
do we want to trim in case of non-email in those methods as well?

Reply:
Done